pkgs/applications/networking/instant-messengers/session-desktop/default.nix

```
@@ -33,7 +33,7 @@ stdenvNoCC.mkDerivation {
       name = "Session";
       desktopName = "Session";
       comment = "Onion routing based messenger";
-      exec = "${appimage}/bin/session-desktop-${version}";
+      exec = "${appimage}/bin/session-desktop";
```

```suggestion
      exec = "session-desktop";
```

See https://github.com/NixOS/nixpkgs/issues/308324